src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py

```diff
@@ -54,15 +54,15 @@ class CCSocketCan(CChannel):
     def __init__(
         self,
         channel: str = "vcan0",
-        remote_id: int = None,
+        remote_id: Optional[int] = None,
```

Comment:
int | None

Reply:
This functionality is available since python 3.10 but pykiso supports python 3.9 : https://github.com/eclipse-kiso-testing/kiso-testing/blob/master/pyproject.toml#L35